curl-sys/build.rs

```diff
@@ -371,20 +371,20 @@ fn try_vcpkg() -> bool {
     }
 
     if successful_probe_details.is_some() {
-        // found libcurl which depends on openssl, libssh2 and zlib
+        // Found libcurl which depends on openssl, libssh2 and zlib
+        // in the a default vcpkg installation. Probe for them
+        // but do not fail if they are not present as we may be working
+        // with a customized vcpkg installation.
         vcpkg::Config::new()
```

Comment:
So, we will have the curl itself here. As for rest of libraries (zlib, ssh2, openssl) - it is up to libz-sys, ssh2-sys and openssl-sys business.

Sounds reasonable.